### ISSUE TITLE
 Error inflating class ImageView for 5.1.0 

### ISSUE THREAD

**Issue by kuzdu** [onfido/onfido-android-sdk#86]
Hi,

#### What was the expected behaviour?
I did take a picture of the back of my driver's license. The next step should be the "Take a selfie"-Screen. 

#### What happened instead?
Layout crashs. (See LogCat output)

#### Version info:

- `onfido-capture-sdk` version: 5.1.0
- Android API version(s): 6.0.1
- `compileSdkVersion`: 28
- `targetSdkVersion`: 28
- `minSdkVersion`: 21
- Support library version (if applicable): androidX
- Device: LGE Nexus 5 

#### Steps to reproduce:
Technical info: I still use the deprecated "Mobile Token SDK" Flow.  Just create a `OnfidoConfig` with `token`, `applicant id` and `capture screen step`   
On device: Take a front picture of a non driver license. Choose "my license is readable". Repeat because nothing is readable (like expected). Took picture again. Choose "my license is readable". Now you have to take a picture of the back of your driver license. Choose "my license is readable" -> Layout crash.       

#### Onfido Flow initialization code:
? 

#### Additional LogCat output:

`E/AndroidRuntime: FATAL EXCEPTION: main
    Process:xxxx PID: 5155
    android.view.InflateException: Binary XML file line #68: Binary XML file line #68: Error inflating class ImageView
        at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at com.onfido.android.sdk.capture.ui.BulletedMessageFragment.onCreateView()
        at com.onfido.android.sdk.capture.ui.FaceIntroFragment.onCreateView()
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2439)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1460)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:802)
        at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625)
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411)
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366)
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273)
        at androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManager.java:733)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: android.view.InflateException: Binary XML file line #68: Error inflating class ImageView
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:782)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
        at com.onfido.android.sdk.capture.ui.BulletedMessageFragment.onCreateView() 
        at com.onfido.android.sdk.capture.ui.FaceIntroFragment.onCreateView() 
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2439) 
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1460) 
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784) 
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852) 
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:802) 
        at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625) 
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411) 
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366) 
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273) 
        at androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManager.java:733) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean androidx.core.content.res.ComplexColorCompat.isStateful()' on a null object reference
        at androidx.vectordrawable.graphics.drawable.VectorDrawableCompat$VFullPath.isStateful(VectorDrawableCompat.java:1946)
        at androidx.vectordrawable.graphics.drawable.VectorDrawableCompat$VGroup.isStateful(VectorDrawableCompat.java:1654)
        at androidx.vectordrawable.graphics.drawable.VectorDrawableCompat$VPathRenderer.isStateful(VectorDrawableCompat.java:1402)
        at androidx.vectordrawable.graphics.drawable.VectorDrawableCompat$VectorDrawableCompatState.isStateful(VectorDrawableCompat.java:1134)
        at androidx.vectordrawable.graphics.drawable.VectorDrawableCompat.isStateful(VectorDrawableCompat.java:525)
        at android.widget.ImageView.updateDrawable(ImageView.java:890)
        at android.widget.ImageView.setImageDrawable(ImageView.java:474)
        at androidx.appcompat.widget.AppCompatImageView.setImageDrawable(AppCompatImageView.java:100)
        at androidx.appcompat.widget.AppCompatImageHelper.loadFromAttributes(AppCompatImageHelper.java:61)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:78)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:68)
        at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:182)
        at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:106)
        at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1266)
        at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1316)
        at android.view.LayoutInflater$FactoryMerger.onCreateView(LayoutInflater.java:189)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:746)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:835) 
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
        at com.onfido.android.sdk.capture.ui.BulletedMessageFragment.onCreateView() 
        at com.onfido.android.sdk.capture.ui.FaceIntroFragment.onCreateView() 
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2439) 
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1460) 
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784) 
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852) 
        at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:802) 
        at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625) 
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411) 
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366) 
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273) 
        at androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManager.java:733) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) `

**Addtional hints**:
_Onfido version 5.0.1 works like expected. Same code._
_With Motorola Moto G 4, Android 7.0, the SDK 5.1.0 works like expected as well._

**Comment by rfreitas** [comment]
Thanks for the info.
Just to confirm you have tested on a LGE Nexus 5 with both 5.0.1 and 5.1.0 and it only crashes on the latter version, correct?

**Comment by kuzdu** [comment]
> Just to confirm you have tested on a LGE Nexus 5 with both 5.0.1 and 5.1.0 and it only crashes on the latter version, correct?

Yes, this is correct.

**Comment by bartzet** [comment]
Hi @kuzdu.

We have run tests internally and with our crowd testing partner with exactly the same device and version and we haven't been able to reproduce.

From the given logs we are not able to tell what the problem might be.

In case you are able to provide us with better logs, please feel free to open a separate issue.

**Comment by mginev** [comment]
This crash is reproduced on android 5 and 6 on onfido sample. Android crashes when trying to inflate ImageView with _onfido_ic_watermark_ drawable. It happenes because _onfido_ic_watermark_ has an empty path which is not allowed in VectorDrawableCompat used by ImageView.
You can even not using onfido sample for reproducing this. Just create an empty project with one activity and add to layout
```
<ImageView
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    app:srcCompat="@drawable/onfido_ic_watermark" />
```
where onfido_ic_watermark is
```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
        android:width="99dp"
        android:height="12dp"
        android:viewportWidth="99.0"
        android:viewportHeight="12.0">
    <path />
</vector>
```
The app will crash on emulator with android 6.0.

**Comment by gokhanaliccii** [comment]
@mginev I couldn't reproduce with android 6.0, do you mind to replace `path` with below sample code to check is it make changes

```
 <path
      android:fillColor="#00FFFFFF" 
      android:pathData=""/>
```

**Comment by mginev** [comment]
@gokhanaliccii It works, no crash with this path

**Comment by gokhanaliccii** [comment]
@mginev Thanks for letting me know, we'll update `onfido_ic_watermark` in the next release

**Comment by gokhanaliccii** [comment]
**Update:**

Instead of 
```
<path
      android:fillColor="#00FFFFFF" 
      android:pathData=""/>
```

We changed with
```
`<path android:fillColor="#00FFFFFF" />`
```

**Comment by mginev** [comment]
@gokhanaliccii Not a great idea. This cause a crash again

**Comment by gokhanaliccii** [comment]
@mginev  Thanks for let me know, reason of that changes was empty `android:pathData` caused a crash on some devices. But after your comment we'll reinvestigate the issue to find better solution

**Comment by bartzet** [comment]
Hi @mginev @kuzdu 
We have released 5.3.1 version today fixing this issue. Please try and let us know if you experience any problems.
